app/Annotations/OpenApi/controllersAnnotations/additionalTrainingAnnotations/AnnotationsAdditionalTraining.php

```diff
@@ -5,9 +5,23 @@ class AnnotationsAdditionalTraining
 {
 /**
  * @OA\Get(
- *     path="/additional-training/list",
+ *     path="/students/{uuid}/additionaltraining",
```

Comment:
If you change the path, remember to tell the frontend people that are applying this endpoints at this moment

Reply:
ok, next task, we'll have it in mind! Thanks!